### PR TITLE
Release/0.9.7

### PR DESCRIPTION
**Pull request by daniely**
@citizen428 If this looks good let's release 0.9.7 and upload to rubygems. Then let's look at supporting minitest 5.

### CODE REVIEW

File: History.txt [hunk 1]
```diff
@@ -1,3 +1,9 @@
+=== 0.9.7 / 2014-03-14
```
Comment (citizen428) [inline History.txt:1]: I'm confused, I see Changelog entries but I don't see commits for the changes they reference (except for fixing `-n`).

Comment (daniely) [inline History.txt:1]: I went through the git log and included things that were added after the 0.9.6 tag. For example, `fix home directory lookup` is 01bba4c4affb329a3096842a2fbde04776ddd3f0

Comment (citizen428) [inline History.txt:1]: :+1: it's pushed.

Comment (daniely) [inline History.txt:1]: Awesome! Merging this into master and closing

**Pull request merged**